splunk_add_on_ucc_framework/global_config_update.py

```diff
@@ -203,3 +205,17 @@ def handle_global_config_update(global_config: global_config_lib.GlobalConfig) -
         _handle_xml_dashboard_update(global_config)
         global_config.dump(global_config.original_path)
         logger.info("Updated globalConfig schema to version 0.0.5")
+
+    _dump_with_migrated_tabs(global_config, global_config.original_path)
+
+
+def _dump_with_migrated_tabs(global_config: GlobalConfig, path: str) -> None:
```

Comment:
If we make a change to the globalConfig - let's indicate it by changing the `schemaVersion` in `meta` in globalConfig file. See examples above.